services/galley/schema/src/V38AddRemoteIdentifiers.hs

```diff
@@ -0,0 +1,29 @@
+module V38AddRemoteIdentifiers (addRemoteIdentifiers) where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+addRemoteIdentifiers :: Migration
+addRemoteIdentifiers = Migration 38 "Add remote identifiers to conversation related tables" $ do
+  -- User table is used to answer the question: What users are part of a
+  -- conversation?  In the context of federation it is now interesting to know:
+  -- Are those users remote or local?
+  schema'
+    [r|
+      ALTER TABLE user ADD (
```

Comment:
This table mostly gives us a list of conversations a user is in. I don't think we need to store that for remote users. We just need a list of users for a conversation.

Reply:
I think what you rather want might be to list the remote conversations a local user is in? So remote_conv instead of remote_user? Did you mix up the changes in the two tables?

Reply:
Reading your comments properly, I really think you mixed up the two tables.

The `member` table maps a conversation to its members/users.
The `user` table maps a user to its conversations.